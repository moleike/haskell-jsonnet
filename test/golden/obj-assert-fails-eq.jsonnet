local x = 'foo';
local y = 'bar';

{ assert true, assert x == y, x: 1 } == { x: 1 }
