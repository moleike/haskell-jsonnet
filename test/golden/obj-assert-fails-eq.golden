Runtime error: Assertion failed
  ./test/golden/obj-assert-fails-eq.jsonnet:4:1-5:1 